src/_setup_packages.sh

```diff
@@ -76,7 +78,7 @@ function install_docker() {
       "apt")
         info "installing docker, this might take some time..."
         _system_cmd_with_debug_and_fail "mkdir -p /etc/apt/keyrings; \
-          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | \
+          wget --output-document - --quiet https://download.docker.com/linux/ubuntu/gpg | \
```

Comment:
maybe use shorthand -O and -q like elsewhere?

Reply:
Done